server/streamable_http.go

```
@@ -0,0 +1,810 @@
+	"sync/atomic"
+
+	"github.com/google/uuid"
+	"github.com/mark3labs/mcp-go/mcp"
+)
+
+// streamableHTTPSession represents an active Streamable HTTP connection.
+type streamableHTTPSession struct {
+	sessionID           string
+	notificationChannel chan mcp.JSONRPCNotification
+	initialized         atomic.Bool
+	lastEventID         string
```

golangci-lint (1.64.8)

field `lastEventID` is unused

(unused)